nix/nixpkgs.nix

```diff
@@ -4,5 +4,11 @@ let
     url = "https://github.com/NixOS/nixpkgs/archive/${srcDef.rev}.tar.gz";
     sha256 = srcDef.sha256;
   };
+  # The version of nixpkgs-fmt in release-19.09 [1] does not support --check, which
+  # we need for CI. Hence this overlay.
+  # [1] https://github.com/NixOS/nixpkgs/blob/release-19.09/pkgs/tools/nix/nixpkgs-fmt/default.nix
```

Comment:
There may well be a better way to get a newer version of `nixpkgs-fmt` than what I've done here. Suggestions welcome :)